src/test/evo_deterministicmns_tests.cpp

```diff
@@ -606,6 +611,7 @@ BOOST_FIXTURE_TEST_CASE(dip3_verify_db, TestChainDIP3Setup)
     BOOST_CHECK_EQUAL(block->GetHash(), ::ChainActive().Tip()->GetBlockHash());
 
     CProRegTx payload;
+    payload.nVersion = bls::bls_legacy_scheme ? CProRegTx::LEGACY_BLS_VERSION : CProRegTx::BASIC_BLS_VERSION;
```

Comment:
I think better to have a helper function such as `CProGRegTx::getVersion(bls::bls_legacy_scheme)` rather that every time write whole condition "bls::bls_legacy_scheme ? CProRegTx::LEGACY_BLS_VERSION : CProRegTx::BASIC_BLS_VERSION;".

It is not only look overexceeding, but may be pain in case if version will be updated once again.